bigtable/google/cloud/bigtable/cluster.py

```diff
@@ -49,7 +44,8 @@ def _prepare_create_request(cluster):
         parent=cluster._instance.name,
         cluster_id=cluster.cluster_id,
         cluster=data_v2_pb2.Cluster(
-            serve_nodes=cluster.serve_nodes,
+            location=cluster.location,
+            serve_nodes=cluster.serve_nodes
```

Comment:
This seems like a bugfix separate from this PR. Can you split it up so it is very clear?